Main.java

```diff
@@ -9,7 +11,38 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+    // create a scanner for user input
+    Scanner input = new Scanner(System.in);
+
+    // to ask the user for the first x coordinate
+    System.out.println("Please enter in the x coordinate of the first point");
+
+    // allows the user to input the x coordinate of the first line
+    int firstX = input.nextInt();
+
+    // asks the user for the first y coordinate
+    System.out.println("Please enter in the y coordinate of the first point");
+
+    // allows the user to input the y coordinate of the first line
+    int firstY = input.nextInt();
+
+    // asks the user for the second x coordinate
+    System.out.println("Please enter in the x coordinate of the second point");
+
+    // allows the user to input the x coordinate of the second line
+    int secondX = input.nextInt();
+
+    // asks the user for the second y coordinate
+    System.out.println("Please enter in the y coordinate of the second point");
+
+    // allows the user to input the y coordinate of the second line
+    int secondY = input.nextInt();
+
+    // calculates the slope of the four points
+    int slope = (secondY - firstY) / (secondX - firstX);
+
```

Comment:
If you look at the sample, the output shows a 2.0. The reason for this is doubles are used instead of ints. Why? Because we can have slopes of things like 2.5 Otherwise, everything else looks good.